simulator/oneshotimporter/importer.go

```diff
@@ -31,6 +31,7 @@ type Service struct {
 var DefaultGVRs = []schema.GroupVersionResource{
 	{Group: "", Version: "v1", Resource: "namespaces"},
 	{Group: "scheduling.k8s.io", Version: "v1", Resource: "priorityclasses"},
+	{Group: "", Version: "v1", Resource: "serviceaccounts"},
```

Comment:
Could you add it also to syncer's DefaultGVRs?

Reply:
done

Reply:
You may forget to push it. `syncer` is not fixed now.